addons/utm/tests/test_utm.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+class TestUtm(odoo.tests.HttpCase):
```

Comment:
Side note: we could probably merge with the small test added in test_utm_consistency.py ? That way we avoid too much test files for this small module.

Side question: why HttpCase ?

Reply:
good question, a bad copy / paste I guess, I change that